Nomad_VanilaJS/Nomad_VanilaJS_JeongBoWoon/js/back ground.js

```diff
@@ -0,0 +1,7 @@
+
+const imgs = ['0.jpg','1.png','2.png'];
+const selectedimg = imgs[Math.floor(Math.random()*imgs.length)];
+
+const bgimg = document.createElement("img");
+bgimg.src = `image/${selectedimg}`;
+document.body.appendChild(bgimg);
```

Comment:
css 속성으로 사용한다면 다음과 같이 사용하면 될 것 같아요!
```suggestion
document.body.style.backgroundImage = `url(img/${selectedImg})`;
```